composer.json

```diff
@@ -68,6 +68,9 @@
     ],
     "test": [
       "phpunit"
+    ],
+    "coverage": [
+      "php -d xdebug.mode=coverage vendor/bin/phpunit --coverage-html tests/coverage"
```

Comment:
Should we migrate CI pipelines to actually track coverage (e.g. run Sonarqube analysis explicitly after test execution) or would that be too frustrating at the moment? 🙈

Reply:
SonarCloud/SonarCube would offer Quality gates also which I like. I see this here as on top. To get it quickly in your local development.

Reply:
"New code" can be pretty much everything when refactoring, reformatting or introducing new hierarchies though, especially on small projects.

We should enable analysis in a follow-up change to not block this one here.